server/routes/search/search.controller.js

```diff
@@ -261,10 +261,10 @@
   };
 
   function flattenStaffList(staffList) {
-    return staffList.reduce((list, curr) => {
+    return staffList ? staffList.reduce((list, curr) => {
       list.push(curr.name);
       return list;
-    }, []).join(',');
+    }, []).join(',') : '';
```

Comment:
should this not return an array instead of `''`?

Reply:
I thought it would return '' since it returns a comma separated string instead of an array when there is data?
The nunjuck file seems to expect a string.

Reply:
that makes sense 😄